src/cloudant/client.py

```diff
@@ -55,11 +55,11 @@ class CouchDB(dict):
 
     def __init__(self, user, auth_token, admin_party=False, **kwargs):
         super(CouchDB, self).__init__()
-        self._cloudant_user = user
-        self._cloudant_token = auth_token
-        self._cloudant_session = None
-        self.cloudant_url = kwargs.get('url')
-        self._cloudant_user_header = None
+        self._user = user
+        self._auth_token = auth_token
+        self._client_session = None
+        self.client_url = kwargs.get('url')
```

Comment:
I can kind of see both arguments (although I think I'd be biased towards calling it `server_url` rather than `client_url`), but maybe just calling it `url` would work?